CHANGELOG.md

```diff
@@ -14,6 +14,13 @@
 - Added Travis CI config for pylint and pyflakes.
 - Added unittest scaffolding with nose
 - Added `AssignerTestCase`, tests for `assigner`, `assigner get`, and all `--help` uses.
+- Added `BackendBase`, `RepoBase`, `StudentRepoBase`, and `TemplateRepoBase`.
```

Comment:
Can you move these lines up under the `Devel` header?

Reply:
I will after you merge the stuff that goes before this :D. Is it merged?

Reply:
This is waiting on #111 which is waiting for me to do the 1.2.0 release (#122) first 😃